WooCommerce/Classes/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardViewModel.swift

```diff
@@ -247,6 +255,28 @@ private extension BlazeCampaignDashboardViewModel {
             .sink { [weak self] _ in
                 self?.analytics.track(event: .Blaze.blazeEntryPointDisplayed(source: .myStoreSection))
             }
+            .store(in: &subscriptions)
+
+        userDefaults.publisher(for: \.hasDismissedBlazeSectionOnMyStore)
+            .dropFirst() // ignores first event because data is already loaded initially.
+            .map { [weak self] hasDismissed -> Bool in
```

Comment:
nit: Can we remove `hasDismissed` to show that we are using that value? 
```suggestion
            .map { [weak self] _ -> Bool in
```

Reply:
Updated in dd908e4b896a90144fcd9c35d12fe257856adb53.